ietf/secr/sreq/views.py

```diff
@@ -33,6 +34,8 @@
 # TODO: This needs to be replaced with something that pays attention to groupfeatures
 AUTHORIZED_ROLES=('WG Chair','WG Secretary','RG Chair','IAB Group Chair','Area Director','Secretariat','Team Chair','IRTF Chair','Program Chair','Program Lead','Program Secretary', 'EDWG Chair')
 
+INFLECT = inflect.engine()
```

Comment:
This forces more start up time delay - that code will be executed as the file is parsed. Sharing it as a global this way has other perils. Is the startup time so long that it cant just be called in the view?

Reply:
yeah - this is tiny, please move it to the view(s) that need it.
```
>>> timeit.timeit(stmt="e=inflect.engine();e.a('history')",setup="import inflect",number=100000)
1.6156303330000128
```